src/schemas/src/dataset-schema.ts

```diff
@@ -100,14 +101,40 @@ export const propertiesV1 = {
   disableDataOperation: null
 };
 
+/**
+ * Prepare a data container for export as part of json / html.
+ * 1) Arrow tables can store Timestamps as BigInts, so convert numbers to ISOStrings compatible with Kepler.gl's TIMESTAMP.
+ * @param dataContainer A data container to flatten.
+ * @returns Row based data.
+ */
+const getAllDataForSaving = (dataContainer: DataContainerInterface): any[][] => {
+  const allData = dataContainer.flattenData();
+
+  if (dataContainer instanceof ArrowDataContainer) {
+    const numColumns = dataContainer.numColumns();
+
+    for (let columnIndex = 0; columnIndex < numColumns; ++columnIndex) {
+      const column = dataContainer.getColumn(columnIndex);
+      // TODO why arrow.Type.Timestamp is undefined?
+      if (column.type.typeId === 10) {
```

Comment:
Can we use ‘ arrow.Type.Timestamp’ enum instead of 10 here? I remember timestamp is not handled well in our initial implementation since it has timestamp, timestamp second, timestamp nanosecond etc.